### PR TITLE
blender: update to 2.79b.

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

**Comment by jnbr** [comment]
Travis doesn't like the amount of warnings spilled by the blender build. Local builds for x86_64, x86_64-musl and i686 succeed.

File: srcpkgs/blender/template [hunk 1]
```diff
@@ -15,7 +15,7 @@ maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="GPL-2"
 homepage="http://blender.org"
```
Comment (cr6git) [inline srcpkgs/blender/template]: `https://www.blender.org`

**Comment by cr6git** [comment]
In [source/](http://download.blender.org/source/) there is a new version dated yesterday:
````
blender-2.79b.tar.gz | 2018-Mar-23
````

**Comment by jnbr** [comment]
I'll update this PR to `2.79b` when there is a official release for it.

**Comment by jnbr** [comment]
> I'll update this PR to 2.79b when there is a official release for it.

Done

**Pull request closed without merging**